configs/modules/t31l_sc2336_atbm6031_defconfig

```diff
@@ -3,3 +3,6 @@ BR2_SOC_INGENIC_T31L=y
 BR2_SENSOR_MODEL="sc2336"
 BR2_PACKAGE_WIFI=y
 BR2_PACKAGE_WIFI_ATBM6031=y
+CONFIG_KERNEL_VERSION_4=y
```

Comment:
lets do that it in an experimental config. we do not want to break the working 3.10 for now.

Reply:
Opps, good catch